.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 
 script:
   - docker run -it --rm -v $PWD:/srv/jekyll -eJEKYLL_UID=$UID jekyll/builder:pages jekyll build --config _config.yml,_prod.yml
-  - docker run -it  --rm -v $(pwd)/_site:/site -it jekyll/builder:pages /usr/gem/bin/htmlproofer /site --disable-external
+  - docker run -it  --rm -v $(pwd)/_site:/site -it jekyll/builder:pages /usr/gem/bin/htmlproofer /site --only_4xx --http-status-ignore "429"
```

Comment:
Why this is `!429` might need listing elsewhere? (comment?)

Reply:
:+1: it actually took me 30s to remember. Commented in 5b25915